package.json

```diff
@@ -46,8 +46,8 @@
     "loader-utils": "^1.1.0",
     "mdx-constant": "^0.1.0",
     "mdx-live": "2.0.0-alpha.2",
-    "next": "7.0.2",
     "next-compose-plugins": "2.1.1",
+    "@primer/next-pages": "0.0.3",
```

Comment:
I believe @primer/next-pages is a devDependency because it's only required in our config setup. Putting it in dependencies causes non-next sites importing blueprints to crash

Reply:
Schwoops nevermind, I forgot it's not a config plugin anymore 🤔